akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,65 @@
+jQuery(document).ready(function ($) {
+  initOldVersionWarnings($);
+});
+
+function initOldVersionWarnings($) {
+  $.get("http://doc.akka.io/docs/versions.json", function (akkaVersionsData) {
+    var thisVersion = browsedAkkaVersion();
+    if (thisVersion.includes("-SNAPSHOT")) {
+      console.log("Detected SNAPSHOT Akka version...");
+      // we could show a 'hey, this is a snapshot' notice
+    } else {
+      for (var series in akkaVersionsData) {
+        if (browsedAkkaVersion().startsWith(series)) {
+          showVersionWarning(thisVersion, akkaVersionsData[series]);
+        }
+      }
+    }
+  });
+}
+
+
+function showVersionWarning(version, seriesInfo) {
+  console.log("Akka version:", version);
+  var targetUrl = ("" + window.location).replace(version, seriesInfo.latest);
+  var $floatyWarning = $('<div id="floaty-warning"/>');
+
+  $floatyWarning
+      .html('' +
+          'You are browsing the docs for Akka ' + version + ', ' +
+          'however the latest release in this series is: ' +
+          '<strong><a href="' + targetUrl + '">' + seriesInfo.latest + '</a></strong>. <br/>' +
+          '<a href="' + targetUrl + '">' +
+          '<string>Click here to go to the latest ' + seriesInfo.latest + ' version of the docs.</strong>' +
+          '</a>');
+
+  if (!!seriesInfo['outdated']) {
+    var instead = seriesInfo['instead'];
+    var unsupportedMessage = '<p style="font-weight: bold">' +
+        'This version of Akka (' + version + ') has been end-of-lifed and is currently unsupported! ' +
+        'Please upgrade to ' + instead + ' (or later) as soon as possible.' +
+        '</p>';
+    $floatyWarning.prepend(unsupportedMessage);
+  }
+
+  $floatyWarning
+      .hide()
+      .prependTo("body")
+      .fadeIn()
+      .delay(10 * 1000)
+      .fadeOut()
+}
+
+// e.g. "docs/akka/2.3.10/scala/persistence.html" => "2.3.10"
+function browsedAkkaVersion() {
+  var fromWindow = window.release;
+  if (fromWindow) {
+    return fromWindow;
+  } else {
+    var path = window.location.pathname;
+    var path2 = path.substring(path.indexOf("2")); // quite hacky
+    var version = path2.substring(0, path2.indexOf("/"));
```

Comment:
Maybe a bit better error handling here so that it doesn't throw an exception that could cause havoc with other stuff in a page if it something unexpected changes with the paths of the docs?